code/datums/shuttles/whiteship.dm

```diff
@@ -0,0 +1,63 @@
+/datum/map_template/shuttle/whiteship
+	port_id = "whiteship"
+
+/datum/map_template/shuttle/whiteship/box
+	suffix = "box"
+	name = "Hospital Ship"
+	description = "Whiteship with medical supplies. Zombies do not currently spawn corpses, and are not infectious."
+
+/datum/map_template/shuttle/whiteship/meta
+	suffix = "meta"
+	name = "Salvage Ship"
+	description = "Whiteship that focuses on a large cargo bay that players can build in. Spawns with Syndicate mobs who do not drop corpses and are highly aggressive."
+
+/datum/map_template/shuttle/whiteship/pubby
+	suffix = "pubby"
+	name = "NT Science Vessel"
+	description = "A small science vessel that uses just one area and is full of angry ants."
+
+/datum/map_template/shuttle/whiteship/cere
+	suffix = "cere"
+	name = "NT Heavy Salvage Vessel"
+	description = "A beefy, well-rounded salvage vessel with a pair of corpses (miner and engineer) and a Captain's hat. Equipped with solar sails and a PACMAN generator."
+
+/datum/map_template/shuttle/whiteship/birdshot
+	suffix = "birdshot"
+	name = "NT Patrol Bee"
+	description = "A small patrol vessel with a central corridor connecting all rooms. Features 2 small cargo bays and a brig. Spawns with an agressive and deadly Gelatinous Cube"
+
+/datum/map_template/shuttle/whiteship/kilo
+	suffix = "kilo"
+	name = "NT Mining Shuttle"
+	description = "A mining vessel with a curious shape starting with a few angry netherworld mobs."
+
+/datum/map_template/shuttle/whiteship/donut
+	suffix = "donut"
+	name = "NT Long-Distance Bluespace Jumper"
+	description = "A ship hit with an engine blowout, leaving it as a depressurised husk. Has infinite power, although likely to bait people into removing that property. Also the most open out of all the whiteships, and starts with a 25% ripley chance."
+
+/datum/map_template/shuttle/whiteship/tram
+	suffix = "tram"
+	name = "NT Long-Distance Bluespace Freighter"
+	description = "A long shuttle that starts with Nanotrasen private security corpses. DOES NOT FIT IN THE BASE DOCKS! Does fit in Deep Space's dock though."
+
+/datum/map_template/shuttle/whiteship/delta
+	suffix = "delta"
+	name = "NT Frigate"
+	description = "A standard whiteship with big spiders onboard. PACMAN generator is not wired and next to main grid cabling, so it requires some work."
+
+/datum/map_template/shuttle/whiteship/pod
+	suffix = "whiteship_pod"
+	name = "Salvage Pod"
+	description = "There is no map for this vessel and it was supposed to be used with the Meta-class. Do not try to spawn it!"
```

Comment:
We no longer have the whiteship pod, right? So why are we keeping this template arond?

Reply:
Removed with https://github.com/tgstation/tgstation/pull/77457/commits/5b10fa4b3079926dbf925d6c66135c6ed7bc6b4e

Also went and removed the two useless pod computers from two whiteships and replaced it with tables.  This should give more space to the cockpits.